devito/dimension.py

```diff
@@ -517,45 +521,29 @@ class SteppingDimension(DerivedDimension):
 
     @property
     def symbolic_start(self):
-        """
-        The symbol defining the iteration start for this dimension.
-
-        note ::
-
-        Internally we always define symbolic iteration ranges in terms
-        of the parent variable.
-        """
         return self.parent.symbolic_start
 
     @property
     def symbolic_end(self):
-        """
-        The symbol defining the iteration end for this dimension.
-
-        note ::
-
-        Internally we always define symbolic iteration ranges in terms
-        of the parent variable.
-        """
         return self.parent.symbolic_end
```

Comment:
docstring gone?

Reply:
it's identical to the one in the superclass, so it gets inherited too